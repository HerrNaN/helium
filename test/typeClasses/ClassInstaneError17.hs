class X a where
    f :: a -> Int

instance X Int

main = f 3