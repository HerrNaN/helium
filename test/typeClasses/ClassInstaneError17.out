Prelude is up to date
Compiling typeClasses/ClassInstaneError17.hs
(4,10): Warning: The function: f for an instance of: X is not defined and doesn't have a default.
(6,1): Warning: Missing type signature: main :: Int
Compilation successful with 2 warnings
exception: undefined.

trace:
  demanded from "LvmLang.<#"
  demanded from "HeliumLang.showInt"
  demanded from "HeliumLang.$primPutChars"
  demanded from "LvmLang.bindIO"
  demanded from "LvmLang.unsafePerformIO"
